CHANGELOG.md

```diff
@@ -22,6 +22,8 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Bumps `com.diffplug.spotless` from 6.9.1 to 6.10.0
 - Bumps `xmlbeans` from 5.1.0 to 5.1.1
 - Bumps `hadoop-hdfs` from 3.3.3 to 3.3.4
+- Exclude jettison version brought in with hadoop-minicluster. ([#4787](https://github.com/opensearch-project/OpenSearch/pull/4787))
+
```

Comment:
Nitpick, but the original PR doesn't add this extra line so probably better not to add it here to minimize conflicts.

Reply:
Sure